src/missing_prices.py

```diff
@@ -105,26 +126,20 @@ def fetch_tokens_without_prices(dune: DuneAPI) -> list[Token]:
     coins = load_coins()
     print(f"Loaded {len(coins)} coins from Coin Paprika")
 
-    # Fetch tokens orders by descending, popularity
-    print("Getting traded tokens without prices from: https://dune.com/queries/224239")
-    tokens = sorted(
-        # Could be interesting to just get the results without executing.
-        fetch_tokens_without_prices(dune=DuneAPI.new_from_environment()),
-        key=lambda t: t.popularity,
-        reverse=True,
-    )
+    tokens = load_tokens("out/missing-token-prices.csv")
     print(f"Fetched {len(tokens)} traded tokens from Dune without prices")
     found, res = 0, []
     for token in tokens:
-        if token.symbol in coins:
-            possibilities = coins[token.symbol]
-            if len(possibilities) == 1:
-                res.append(token.coin_paprika_rep(possibilities[0]["id"]))
-                found += 1
-            else:
-                print(f"non unique {token}: {len(possibilities)} occurrences")
-        if found > 50:
-            print("Stopped at 50 results.")
-            break
-
-    write_results(results=res, path="./out", filename="missing-prices.yaml")
```

Comment:
I believe you forgot to add this line back, since I don't have `"out/missing-token-prices.csv"` saved when fetching the tokens.

Reply:
Right yes this is in my local history. Sorry about that. I am going to clean this up anyway since dune wants to use it in their backend.

Reply:
However, the PR description did say

> Note that it requires having downloaded the query results of https://dune.com/queries/1317238 and saving them to "out/missing-token-prices.csv".

Reply:
Fixed this to pull from Dune.